modules/navigation/3d/nav_mesh_queries_3d.cpp

```diff
@@ -1154,6 +1120,52 @@ void NavMeshQueries3D::_query_task_clip_path(NavMeshPathQueryTask3D &p_query_tas
 	}
 }
 
+bool NavMeshQueries3D::_query_task_is_connection_owner_usable(const NavMeshPathQueryTask3D &p_query_task, const NavBaseIteration *p_owner) {
+	ERR_FAIL_NULL_V(p_owner, false);
```

Comment:
if we took p_owner by reference we wouldn't need this ?

Reply:
Removed as we don't really need this as the owner was checked before this function is called, else the entire logic would crash way before. When the iterations get changed to ref this part is updated anyway.